packages/rum-core/src/domain/contexts/viewContexts.ts

```diff
@@ -45,6 +46,7 @@ export function startViewContexts(lifeCycle: LifeCycle): ViewContexts {
     return {
       service: view.service,
       version: view.version,
+      customerContext: view.context,
```

Comment:
**💬 suggestion:** ‏‏I'm not a big fan of the `customer` prefix here. Other properties of this interface also come from the customer, so it feels weird to specify `customer` for context.
 
I understand you are using `customerContext` instead of `context` to avoid having a `context` field on an interface that is already called `xxxContext`? I agree that's too many context! But I would rather rename `ViewContext` to something like `ViewHistoryEntry` (as I suggested in the RFC) instead of using `customerContext`.

Reply:
I'm moving this renaming to another PR.